docs/src/main/asciidoc/spring-cloud-netflix.adoc

```diff
@@ -1493,9 +1493,38 @@ The location of the backend can be specified as either a "serviceId"
       url: http://example.com/users_service
 ----
 
-These simple url-routes don't get executed as a `HystrixCommand` nor can you loadbalance multiple URLs with Ribbon.
-To achieve this, specify a service-route and configure a Ribbon client for the
-serviceId (this currently requires disabling Eureka support in Ribbon:
+These simple url-routes don't get executed as a `HystrixCommand` nor do they loadbalance multiple URLs with Ribbon.
+To achieve this, you can specify a "virtual" service:
```

Comment:
I don't know that I would call it a virtual service. It's just a service who's list of servers is static, not dynamic.